lib/registration.pm

```diff
@@ -109,7 +109,7 @@ sub accept_addons_license {
 
     # In SLE 15 some modules do not have license or have the same
     # license (see bsc#1089163) and so are not be shown twice
-    push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless is_sle('15+');
+    push @addons_with_license, @SLE15_ADDONS_WITHOUT_LICENSE unless (is_sle('15+') or (is_upgrade and is_sle('12-sp5+')));
```

Comment:
in perl `or` is really tricky and not always the same as `||` I recommend you to use `||`

Reply:
Pushing changes with the suggestion, although I don't think there's much of a difference between `or` and `||` on properly parenthesized sentences like this one, as their main difference is the lower precedence of `or` ... I always find `or` easier to read. :)

Verification run in http://mango.suse.de/tests/1295